packages/slate-react/src/components/editable.tsx

```diff
@@ -250,10 +250,20 @@ export const Editable = (props: EditableProps) => {
   // The autoFocus TextareaHTMLAttribute doesn't do anything on a div, so it
   // needs to be manually focused.
   useEffect(() => {
-    if (ref.current && autoFocus) {
-      ref.current.focus()
-    }
-  }, [autoFocus])
+      if (ref.current && autoFocus) {
+        if(window.getSelection && document.createRange) {
+          const range = document.createRange();
+          node.focus();
```

Comment:
what is `node`?

Reply:
@akonsu Sorry, node was a reference to ref.current in my other sandbox. Will correct it now.